hooks/usePlaceSearch.ts

```
@@ -0,0 +1,742 @@
+      }
+      // Polygon 표시 또는 중심 이동
+      if (
+        window.kakao &&
+        window.kakao.maps &&
+        window.kakao.maps.services
+      ) {
+        const geocoder = new window.kakao.maps.services.Geocoder()
+        geocoder.addressSearch(value, (result: any[], status: string) => {
+          if (
+            status === window.kakao.maps.services.Status.OK &&
+            result.length > 0
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)